crates/cxx-qt-lib/src/core/qtypes.rs

```diff
@@ -0,0 +1,189 @@
+// SPDX-FileCopyrightText: 2024 Klarälvdalens Datakonsult AB, a KDAB Group company <info@kdab.com>
+// SPDX-FileContributor: Andrew Hayzen <andrew.hayzen@kdab.com>
+//
+// SPDX-License-Identifier: MIT OR Apache-2.0
+
+use cxx::{type_id, ExternType};
+use std::mem::MaybeUninit;
+
+#[cxx::bridge]
+mod ffi {
+    unsafe extern "C++" {
+        include!("cxx-qt-lib/qtypes.h");
+
+        #[cxx_name = "qint64"]
+        type QInt64 = super::QInt64;
```

Comment:
In Rust should we be calling these `QInt64` and `QUInt64` or should we copy Qt and use lowercase `qint64` and `quint64` ? :thinking: